vss-cats/src/main/scala/com/virtuslab/vss/cats/base/http/routes/PasswordRoutes.scala

```diff
@@ -0,0 +1,32 @@
+package com.virtuslab.vss.cats.base.http.routes
+
+import cats.Monad
+import cats.syntax.all.*
+import cats.effect.kernel.Async
+import org.http4s.dsl.Http4sDsl
+import org.http4s.HttpRoutes
+import com.virtuslab.vss.common.*
+import com.virtuslab.vss.cats.base.services.*
+import com.virtuslab.vss.common.BaseEndpoints
+import sttp.tapir.server.ServerEndpoint
+import sttp.tapir.*
+import natchez.Trace
+
+final case class PasswordRoutes[F[_]: Monad: Async: Trace](
+  passwords: Passwords[F]
+) extends Http4sDsl[F]:
+  val routes: List[ServerEndpoint[Any, F]] =
+    List(
```

Comment:
Hash endpoint is not working. I tried the basic request in the Swagger and I got 400.
gRPC endpoint is working. Also other HTTP endpoints are working.

![image](https://github.com/VirtusLab/vss/assets/50501718/77d87228-59ad-4ea7-a00b-8e122d04357e)

Reply:
Thanks, I must've messed something up when applying previous corrections.